combat-trainer.lic

```diff
@@ -1881,7 +1884,7 @@ class GameState
       echo('check_charging?: done') if $debug_mode_ct
 
       cambrinth_charges(nil)
-      invoke(@cambrinth)
+      invoke(@cambrinth, @dedicated_camb_use)
```

Comment:
Good point. But you'll still have to pass in something or Ruby will complain about the number of arguments.